### ISSUE TITLE
Create a user delegation SAS: pre-2020-02-10 StringToSign incorrect fields

### ISSUE THREAD

**Issue by buckelij** [MicrosoftDocs/feedback#3445]
**Describe the bug**

I've identified two problems with the [Create a user delegation SAS](https://docs.microsoft.com/en-us/rest/api/storageservices/create-user-delegation-sas) document:

* The document states "All features added with version 2020-02-10 are currently in preview. The preview is intended for non-production use only." and says in various places throughout the document that 2020-02-10 features are in preview. Based on https://docs.microsoft.com/en-us/rest/api/storageservices/versioning-for-the-azure-storage-services I believe 2020-04-08 is actually the most recent production version.
* The [Specify the signature](https://docs.microsoft.com/en-us/rest/api/storageservices/create-user-delegation-sas#specify-the-signature) "StringToSign" for versions prior to 2020-02-10 has the fields wrong.

The example is:
> Use the following format for the string-to-sign:
```
StringToSign = sp + "\n" +  
 st + "\n" +  
 se + "\n" +  
 canonicalizedResource + "\n" +  
 skoid + "\n" +
 sktid + "\n" +
 skt + "\n" +
 ske  + "\n" +
 sks + "\n" +
 skv + "\n" +
 saoid + "\n" +
 suoid + "\n" +
 scid + "\n" +
 sip + "\n" +  
 spr + "\n" +  
 sv + "\n" +  
 sr + "\n" +
 rscc + "\n" +
 rscd + "\n" +  
 rsce + "\n" +  
 rscl + "\n" +  
 rsct
```

The format that actually works for 2019-12-12 is:

```
StringToSign = sp + "\n" +  
 st + "\n" +  
 se + "\n" +  
 canonicalizedResource + "\n" +  
 skoid + "\n" +
 sktid + "\n" +
 skt + "\n" +
 ske  + "\n" +
 sks + "\n" +
 skv + "\n" +
 sip + "\n" +  
 spr + "\n" +  
 sv + "\n" +  
 sr + "\n" +
 "\n" +
 rscc + "\n" +
 rscd + "\n" +  
 rsce + "\n" +  
 rscl + "\n" +  
 rsct
```

In other words, omit  `saoid + "\n" + suoid + "\n" + scid + "\n"`, which are not relevant to 2019-12-12, and add a blank line after `sr`. That blank line lines up with the `signedSnapshotTime` field for the 2020-02-10 or higher version, but there's no reference to that field for earlier versions.

**To Reproduce**

Generate a user delegation SAS with version 2019-12-12 and find that the required StringToSign format is not as documented. 

**Expected behavior**
The documentation should describe the correct StringToSign format and should indicate 2020-04-08 is the most recent production api version.

**Additional context**
I can share code demonstrating the problem in a non-public forum.

**Comment by rootsmusic** [comment]
@buckelij Azure REST API specs are at https://github.com/Azure/azure-rest-api-specs
